templates/mod/disk_cache.conf.erb

```diff
@@ -2,3 +2,6 @@ CacheEnable disk /
 CacheRoot "<%= @_cache_root %>"
 CacheDirLevels 2
 CacheDirLength 1
+<% if scope['cache_ignore_headers'] != :undef -%>
```

Comment:
I think you can simply use ```<% if @cache_ignore_headers -%>``` here

Reply:
I'd looked into this before I use the scope:
https://stackoverflow.com/questions/17096934/check-if-a-variable-is-undef-in-puppet-template

It looks like by simply using if is not sufficient to know that a variable is really undef.

Reply:
It's good to note that that answer is from 2013. Puppet has evolved a lot so I'm pretty sure that's no longer needed.